tb/ariane_tb.cpp

```diff
@@ -19,7 +19,7 @@
 #include "verilator.h"
 #include "verilated.h"
 #include "verilated_vcd_c.h"
-#include "Variane_testharness__Dpi.h"
+//#include "Variane_testharness__Dpi.h"
```

Comment:
Why is this needed?

Reply:
Opps.  Good catch, thanks.  That should not have been pushed in.  Restored.